src/server/pfs/cmds/cmds.go

```diff
@@ -624,11 +569,101 @@ $ {{alias}} test@master --new`,
 	subscribeCommit.Flags().StringVar(&from, "from", "", "subscribe to all commits since this commit")
 	subscribeCommit.MarkFlagCustom("from", "__pachctl_get_commit $(__parse_repo ${nouns[0]})")
 	subscribeCommit.Flags().BoolVar(&newCommits, "new", false, "subscribe to only new commits created from now on")
-	subscribeCommit.Flags().AddFlagSet(rawFlags)
-	subscribeCommit.Flags().AddFlagSet(fullTimestampsFlags)
+	subscribeCommit.Flags().AddFlagSet(outputFlags)
+	subscribeCommit.Flags().AddFlagSet(timestampFlags)
 	shell.RegisterCompletionFunc(subscribeCommit, shell.BranchCompletion)
 	commands = append(commands, cmdutil.CreateAlias(subscribeCommit, "subscribe commit"))
 
+	writeCommitTable := func(commitInfos []*pfs.CommitInfo) error {
+		if raw {
+			encoder := cmdutil.Encoder(output, os.Stdout)
+			for _, commitInfo := range commitInfos {
+				if err := encoder.EncodeProto(commitInfo); err != nil {
+					return err
+				}
+			}
+			return nil
+		} else if output != "" {
+			return errors.New("cannot set --output (-o) without --raw")
+		}
+
+		return pager.Page(noPager, os.Stdout, func(w io.Writer) error {
+			writer := tabwriter.NewWriter(w, pretty.CommitHeader)
+			for _, commitInfo := range commitInfos {
+				pretty.PrintCommitInfo(writer, commitInfo, fullTimestamps)
+			}
+			return writer.Flush()
+		})
+	}
+
+	waitCommitSet := &cobra.Command{
+		Use:   "{{alias}} <commitset-id>",
+		Short: "Return info about the commits in a commitset.",
+		Long:  "Return info about the commits in a commitset.",
+		Run: cmdutil.RunFixedArgs(1, func(args []string) error {
+			toBranches := []*pfs.Branch{}
+			for _, arg := range branches {
+				branch, err := cmdutil.ParseBranch(arg)
+				if err != nil {
+					return err
+				}
+				toBranches = append(toBranches, branch)
+			}
+
+			client, err := client.NewOnUserMachine("user")
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			var commitInfos []*pfs.CommitInfo
+			if len(toBranches) != 0 {
+				for _, branch := range toBranches {
+					ci, err := client.WaitCommit(branch.Repo.Name, branch.Name, args[0])
```

Comment:
Oh huh, didn't realize this wasn't part of the gRPC

Reply:
It's just a porcelain function around `InspectCommit` with `wait=true`